generators/ci-cd/prompts.js

```diff
@@ -16,139 +16,220 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const chalk = require('chalk');
+
 module.exports = {
-    askPipelines,
+    askPipeline,
     askIntegrations
 };
 
-function askPipelines() {
+function askPipeline() {
     if (this.abort) return;
     if (this.autoconfigureTravis) {
         this.log('Auto-configuring Travis CI');
-        this.pipelines = ['travis'];
+        this.pipeline = 'travis';
         return;
     }
     if (this.autoconfigureJenkins) {
         this.log('Auto-configuring Jenkins');
-        this.pipelines = ['jenkins'];
+        this.pipeline = 'jenkins';
+        this.sendBuildToGitlab = false;
+        this.insideDocker = false;
         return;
     }
     const done = this.async();
     const prompts = [
         {
-            type: 'checkbox',
-            name: 'pipelines',
+            type: 'list',
+            name: 'pipeline',
             message: 'What CI/CD pipeline do you want to generate?',
-            default: [],
+            default: 'jenkins',
             choices: [
                 { name: 'Jenkins pipeline', value: 'jenkins' },
-                { name: 'Travis CI', value: 'travis' },
                 { name: 'GitLab CI', value: 'gitlab' },
-                { name: 'CircleCI', value: 'circle' }
+                { name: 'Travis CI', value: 'travis' }
             ]
         }
     ];
     this.prompt(prompts).then((props) => {
-        if (props.pipelines.length === 0) {
+        if (props.pipeline.length === 0) {
             this.abort = true;
         }
-        this.pipelines = props.pipelines;
+        this.pipeline = props.pipeline;
         done();
     });
 }
 
 function askIntegrations() {
-    if (this.abort || this.pipelines.length === 0) return;
+    if (this.abort || this.pipeline === undefined) return;
```

Comment:
i think `if (this.abort || !this.pipeline) return;` is enough

Reply:
right, I will change this